test/unit/org/apache/cassandra/db/CleanupTest.java

```diff
@@ -117,9 +117,11 @@ public String getDatacenter(InetAddressAndPort endpoint)
     }
 
     @Test
-    public void testCleanup() throws ExecutionException, InterruptedException
+    public void testCleanup() throws ExecutionException, InterruptedException, UnknownHostException
```

Comment:
isnt just `Throwable` enough? It will not hurt anything, if it fails, it fails.

Reply:
I think it's fine to keep as is since the other tests follow the same pattern

Reply:
Right ... interesting. @pauloricardomg do you think this might be just `Throwable` and we fix that at all other places? I think it is worth it and the change is cosmetic. I consider these explicitly enumerated exceptions for other test methods in this class quite unusual to be there ... I dont know I have ever seen it like that in Cassandra tests.

Reply:
I don't really mind and I think it's fine to keep it as it is since this is an old test, to keep the scope of this change small. But if we were to change I'd normally prefer `throws Exception` since this is usually sufficient to catch checked exceptions.